tests/render.spec.js

```diff
@@ -61,4 +65,8 @@ describe('render', () => {
     expect(getByText('Hello world!')).toBeInTheDocument()
     expect(getByTestId('custom-target')).toBeInTheDocument()
   })
+
+  test('after each test above, document is clean from targets and components', () => {
+    expect(document.body.innerHTML).toBe('<div></div>')
```

Comment:
output here is dependent on previous tests,

- before: `<div></div><div></div><div></div><div></div><main data-testid=\"custom-target\"></main>`

Reply:
I think the body will be entirely empty after this great addition.

Reply:
I agree with @emiltholin. It should be empty

Reply:
indeed is happening, as we can see in the failed build on travis:

- https://travis-ci.org/testing-library/svelte-testing-library/builds/528192604#L503

i bet it was some kind of local jest cache, running with `--no-cache` seems to fixe it!

thanks @EmilTholin @benmonro